src/components/navbar/Navbar.jsx

```diff
@@ -2,21 +2,25 @@ import React from "react";
 import { AiFillLike } from "react-icons/ai";
 import { BiTimeFive } from "react-icons/bi";
 import { BsSearch } from "react-icons/bs";
-import { FaUserCircle, FaHistory } from "react-icons/fa";
+import { FaHistory, FaUserCircle } from "react-icons/fa";
 import { GiHamburgerMenu } from "react-icons/gi";
 import { HiOutlineLogout } from "react-icons/hi";
 import { MdExplore, MdPlaylistPlay } from "react-icons/md";
 import { Link, useNavigate } from "react-router-dom";
 
-export const Navbar = () => {
+export const Navbar = (props) => {
+  const { setSearchValue } = props;
   const navigate = useNavigate();
   const logoutHandler = () => {
```

Comment:
Fix: In the case of logout you also need to remove the data of the user that you have saved in the context.

Reply:
ok i'll do that